ConfigurationSystem/Agent/VOMS2CSAgent.py

```diff
@@ -81,7 +81,7 @@ def execute( self ):
     return S_OK()
 
   @executeWithUserProxy
-  def __syncCSWithVOMS( self, vo ):
+  def __syncCSWithVOMS( self, vo, proxyUserName=None, proxyUserGroup=None ):
```

Comment:
As @atsareg said, these two parameters are used by the decorator. In fact they are even poped from kwargs, so they will never make it to the method. And this is why there was the pylint helper above.